### ISSUE TITLE
[BUG] Metrics on power badge dont match email body

### ISSUE THREAD

**Issue by RickNY** [cagnulein/qdomyos-zwift#1564]
**Describe the bug**
After doing a treadmill Peloton workout on QZ, the Power Badge metrics do not match the email body metrics - or display NaN

**To Reproduce**
1. In iOS 2.13.95, started treadmill workout along with Apple Watch
2. At completion of workout - check emailed report
3. Compare to Power Badge image -- Distance differs, AVG Resistance displays NaN IvI

**Expected behavior**
Metrics should match, metrics that were not collected should display N/A or be blank instead of NaN

**Smartphone (please complete the following information):**
 - iPhone 14 Pro
 - ios 16.5.1(a)

**Append a debug log**

 
[debug-Tue_Jul_11_17_27_54_2023.log.txt](https://github.com/cagnulein/qdomyos-zwift/files/12027866/debug-Tue_Jul_11_17_27_54_2023.log.txt)
![email_report](https://github.com/cagnulein/qdomyos-zwift/assets/702933/8470c40c-4a2f-48a5-862f-6c06da84a4b0)
![Tue Jul 11 17_50_07 2023_power_badge](https://github.com/cagnulein/qdomyos-zwift/assets/702933/a9c83f2b-8040-4379-90c6-ad535f594bed)

**Comment by cagnulein** [comment]
Yes distance it's rounder to the integer and resistance of course it's not applicable for treadmills 

I will replace it with inclination or speed
